src/python/WMCore/FwkJobReport/Report.py

```diff
@@ -1323,6 +1323,20 @@ def setAcquisitionProcessing(self, acquisitionEra, processingVer, processingStr=
 
         return
 
+    def updateSubprocessInfo(self, sysTime, userTime, startTime, endTime):
+        """
+        Add process information to WMCore FJR
```

Comment:
Please improve this docstring (with better explanation and up-to-date parameters)

Reply:
done